packages/onboarding/src/components/WooPaymentGatewayConfigure/README.md

```diff
@@ -6,7 +6,13 @@ A Slotfill component that will replace the <DynamicForm /> component involved in
 
 ```jsx
 <WooPaymentGatewayConfigure id={ key }>
-  {({defaultForm: DefaultForm}) => <p>Fill Content</p>}
+  {({defaultForm: DefaultForm}) => {
+    return <>
+    <p>Fill Content
```

Comment:
Thanks for updating this 🎉 

The formatting is a little off here, but awesome to expand on this example!